src/App.java

```diff
@@ -1,34 +1,264 @@
+            String place = getValidInput();
+
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj3 = getValidInput();
+
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun2 = getValidInput();
+
+            System.out.print(g+"Emotion:\n"+w+" ↳ ");
+            String emotion1 = getValidInput();
+
+            // print out madlibs
+            System.out.println(b+"Today, I, " + userName + ", went to the zoo and saw a very " + adj1 + " " + animal1 + ".");
+            System.out.println("It was " + verb_ing1 + " near the " + noun1 + " and looked really " + adj2 + ".");
+            System.out.println("I decided to " + verb1 + " closer, but then it " + ptverb1 + " towards (the) " + place + ".");
+            System.out.println("It was so " + adj3 + "! After that, I bought a " + noun2 + " and left feeling " + emotion1);
+            System.out.println(r+"==============");
+        } else if (choice == 2) { // story 2
+            // collect inputs
+            System.out.print(g+ "Adjective:\n"+w+" ↳ ");
+            String adj1 = getValidInput();
+
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun1 = getValidInput();
+            
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj2 = getValidInput();
+
+            System.out.print(g+"Verb:\n"+w+" ↳ ");
+            String verb1 = getValidInput();
+
+            System.out.print(g+"Animal:\n"+w+" ↳ ");
+            String animal1 = getValidInput();
+
+            System.out.print(g+"Verb:\n"+w+" ↳ ");
+            String verb2 = getValidInput();
+
+            System.out.print(g+"Verb:\n"+w+" ↳ ");
+            String verb3 = getValidInput();
+
+            System.out.print(g+"Place:\n"+w+" ↳ ");
+            String place1 = getValidInput();
+
+            System.out.print(g+"Verb:\n"+w+" ↳ ");
+            String verb4 = getValidInput();
+
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun2 = getValidInput();
+
+            System.out.print(g+"Adjective:\n"+w+" ↳ ");
+            String adj3 = getValidInput();
+
+            System.out.print(g+"Noun:\n"+w+" ↳ ");
+            String noun3 = getValidInput();
+            
+            // print out madlibs
+            System.out.println("In a small, " + adj1 + " " + noun1 + " shop, I found a " + adj2 + " potion on a shelf.");
+            System.out.println("The label said it could make you " + verb1 + " like a " + animal1 + ". I decided to " + verb2 + " it.");
+            System.out.println("Immediately, I started to " + verb3 + " all over (the) " + place1 + ", and suddenly, I could " + verb4 + ".");
+            System.out.println("I kept going until I accidentally knocked over a " + noun2 + ".");
+            System.out.println("I'll never forget that " + adj3 + " day. The day I became (a/an) " + noun3 + ".");
+            System.out.println(r+"==============");
+        } else if (choice == 3) { // story 3
```

Comment:
"else" would probably be better here. When you have a chain of if statements you are almost always best off ending with "else" if you are certain you want one of the elements to run, it can be more optimally compiled for and makes it less likely to get weird runtime errors.